.travis.yml

```diff
@@ -18,5 +18,5 @@ services:
   - postgresql
 
 before_script:
-  - psql -c "CREATE DATABASE bookstest;" -U postgres
+  - psql -c "CREATE DATABASE db_gm_training_express_js;" -U postgres
```

Comment:
if you want you can also include the command for creating the user that you are gonna be connecting with on testing